docs/docs/20-usage/15-terminology/index.md

```diff
@@ -33,16 +33,6 @@
 
 ![Relation between pipelines, workflows and steps](pipeline-workflow-step.svg)
 
-## Pipeline events
```

Comment:
Would be nice to have a link to the new location here

Reply:
I'd say strictly speaking the syntax values aren't terminology, e.g. `pull_request_closed` isn't really a term.

Reply:
we should add an link to L17 @not-my-profile